web/viewer.css

```diff
@@ -321,31 +327,45 @@ body {
   transition-property: inset-inline-start;
 }
 
-.toolbar {
-  position: relative;
-  inset-inline: 0;
-  z-index: 9999;
-  cursor: default;
-  font: message-box;
-}
-
-:is(.toolbar, .editorParamsToolbar, .findbar, #sidebarContainer)
-  :is(input, button, select),
-.secondaryToolbar :is(input, button, a, select) {
-  outline: none;
+#sidebarContainer :is(input, button, select) {
+  /*outline: none;*/
```

Comment:
This should either be uncommented, or the line removed if it's actually unnecessary.